numpy/core/src/common/npy_partition.h.src

```diff
@@ -80,9 +90,11 @@ static part_map _part_map[] = {
         NPY_@TYPE@,
         {
             (PyArray_PartitionFunc *)&introselect_@suff@,
+            (PyArray_PartitionFunc *)&floydselect_@suff@,
```

Comment:
@malyasova If libraries linked against an older version of NumPy try to use this, they won't be able to, because the `_part_map` array now has a different structure and indices than it used to before.